### PR TITLE
Add kafka init image

### LINKED ISSUE #182: Automated docker-build of radarbase-kafka-init
Kubernetes does not support building images from build context. We need an automated way of building `radar-kafka-init` image corresponding to schema changes.

### PR DESCRIPTION
**Pull request by nivemaham**
we should have automated build for kafka-init image. Kubernetes does not support building images from local context. 
This approach combines the building process of radar-schema and how we build kafka-init on RADAR-Docker
Should fix #182

### CODE REVIEW

File: Dockerfile.kafkaInit [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+RUN mkdir -p /schema/merged /schema/java/src /schema/java/classes /usr/share/java
+
+WORKDIR /schema
+
+ENV JQ_VERSION=1.5
+RUN curl -L#o /usr/bin/jq https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 \
+    && chmod +x /usr/bin/jq
+RUN curl -#o /usr/share/java/avro-tools.jar \
+ "$(curl -s http://www.apache.org/dyn/closer.cgi/avro/\?as_json \
+  | jq --raw-output ".preferred")avro/avro-1.8.2/java/avro-tools-1.8.2.jar"
```
Comment (yatharthranjan) [inline Dockerfile.kafkaInit]: Avro 1.8.2 is not not available anymore at that path. It has been archived. Please take a look at https://github.com/RADAR-base/RADAR-Docker/pull/192

File: Dockerfile.kafkaInit [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+COPY java-sdk/radar-schemas-tools/build.gradle /code/java-sdk/radar-schemas-tools/
+COPY java-sdk/radar-schemas-tools/src /code/java-sdk/radar-schemas-tools/src
+RUN ./gradlew distTar && cd radar-schemas-tools/build/distributions && tar xzf radar-schemas-tools*.tar.gz
+
+
+FROM openjdk:8-alpine
+
+ENV KAFKA_SCHEMA_REGISTRY=http://schema-registry-1:8081
+ENV KAFKA_NUM_PARTITIONS=3
+ENV KAFKA_NUM_REPLICATION=2
```
Comment (yatharthranjan) [inline Dockerfile.kafkaInit]: Can we change this to 3? Although it is overridden in docker-compose in RADAR-Docker to 3 still will be nice to have it here as well.

Comment (nivemaham) [reply Dockerfile.kafkaInit]: Do you mean the replication?

File: Dockerfile.kafkaInit (hunk 1 above)
Comment (nivemaham) [reply Dockerfile.kafkaInit]: Sorry. My bad

File: Dockerfile.kafkaInit (hunk 2 above)
Comment (yatharthranjan) [reply Dockerfile.kafkaInit]: Yes. Thank you

File: java-sdk/build.gradle [hunk 3]
```diff
@@ -17,7 +17,7 @@ subprojects {
     apply plugin: 'idea'
 
     // Configuration
-    version = '0.5.3-SNAPSHOT'
+    version = '0.5.3'
```
Comment (yatharthranjan) [inline java-sdk/build.gradle]: Can we change back to SNAPSHOT version since we are merging into dev?

**Pull request merged**